### PR TITLE
Makes Fartium explode when heated

### PR DESCRIPTION
**Pull request by ArcLumin**
Fartium now explodes when heated to 474k

### CODE REVIEW

**Comment by Spacedong** [comment]
only if it makes the superfart noise, DNM until it does

**Comment by ArcLumin** [comment]
Aww spacedong thinks his input matters, how cute


seriously though, I should do that

**Comment by KayeArray** [comment]
Is the explosion limited to 1, that's what the code means right?

**Comment by ArcLumin** [comment]
Nope

**Comment by ArcLumin** [comment]
This now plays the superfart sound when the explosion happens

**Comment by ghost** [comment]
Amazing +1

**Comment by JohnGinnane** [comment]
Have you tested this?

File: html/changelogs/ArcLumin - Something Smells.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+author: ArcLumin
+delete-after: True
+changes: 
+  - tweak: "Fartium now explodes when heated."
```
Comment (JohnGinnane) [inline html/changelogs/ArcLumin - Something Smells.yml]: I would suggesting adding the temperature for info

File: code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm [hunk 2]
```diff
@@ -521,7 +521,22 @@
 				H << "<span class='danger'>Your stomach hurts too much as pressure builds up inside of you.</span>"
 				H.adjustToxLoss(6*REM)
 	..()
-
+  
+/datum/chemical_reaction/fartium_explosion
```
Comment (JohnGinnane) [inline code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm]: I really think this should go into r/code/modules/reagents/Chemistry-Recipes/Pyrotechnics.dm along with other reactions

**Review by JohnGinnane — DISMISSED** [review]

File: code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm (hunk 2 above)
Comment (TheOneAndOnlyCreeperJoe) [reply code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm]: I swear to god if this meme PR gets merged before the admin logging and creates conflicts I'll be mad.

Comment (ArcLumin) [reply code/modules/reagents/Chemistry-Reagents/Drug-Reagents.dm]: I can set a DNM until the admin logging @TheOneAndOnlyCreeperJoe 

Also, yeah, was late night/early morning code, will work on it when I return

**Comment by JohnGinnane** [comment]
Grand so

**Comment by KittyKathy** [comment]
This is just a balancing concern but it may be too powerful of a explosion. I don't know if this is intended or not but it is stronger than Nitroglycerin I think...like twice as strong. So like 60 units of this stuff is a max cap if i calculated that right not sure though. Just for reference this is 6 times stronger than black powder I think.

*Edited for a typo

**Comment by KayeArray** [comment]
@KittyKathy That's what I was afraid of this is literally the most potent explosive in the game.

**Comment by Hackole** [comment]
If this is really more potent than the other explosive chems, I can't really support it. Fartium is super easy to get too, just grind butts

**Comment by ArcLumin** [comment]
And yeah, this was originally a late night meme pr, intended to fail. Now that I'm awake and sober I'll make it not a meme

**Comment by ArcLumin** [comment]
Fartium explosion size has been divided by 8

**Comment by KittyKathy** [comment]
Just a suggestion but maybe it should be stronger than black powder because there is no reason to use this over black powder other than for the meme sound and no delay to explode when heated, which you can bypass the delay black powder has when heated by causing the on_ex_act of it. It is harder to get in large quantities than black powder without some system sort of duplication system, but if you were going to duplicate chemicals to Cuban Pete you would use nitroglycerin anyway.

**Comment by Swagile** [comment]
its also hard to get butts unless you change all monkeys in genetics pen to humans and do surgery / take butts off of people who super fart (unlikely to get enough of it to do anything in time)

so making it be a better explosion than blackpowder would be gr8 due to the effort needed to be put into this that can be done faster by just doing black powder

**Comment by ArcLumin** [comment]
Boosted to /3

**Comment by SpankMasterson** [comment]
Make it a weakish explosion and I like it

**Comment by Swagile** [comment]
ill give my GM + 1 then since you boosted it, because if its just a weakish explosion it will be a meme chem no one will use except for jokes / boredom

**Comment by Hackole** [comment]
Alright, seems fine enough. +1

**Comment by ArcLumin** [comment]
mergan then

**Pull request merged**